src/models/room-state.ts

```diff
@@ -53,7 +53,7 @@ export class RoomState extends EventEmitter {
     private modified: number;
 
     // XXX: Should be read-only
-    public members: Record<string, RoomMember> = {}; // userId: RoomMember
+    public members = new Map<string, RoomMember>(); // userId: RoomMember
```

Comment:
We access this field in a few places so this would need a version bump

![image](https://user-images.githubusercontent.com/2403652/126508401-b2f2d7d2-f6a9-4ae3-9fd0-e6613111883e.png)